S/SFML/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "SFML"
+version = v"2.5.1"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource(
+        "https://github.com/SFML/SFML.git",
+        "2f11710abc5aa478503a7ff3f9e654bd2078ebab",
+    ),
+    ArchiveSource(
+        "https://www.sfml-dev.org/files/SFML-2.5.1-linux-gcc-64-bit.tar.gz",
+        "34ad106e4592d2ec03245db5e8ad8fbf85c256d6ef9e337e8cf5c4345dc583dd",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# build SFML
+cd ${WORKSPACE}/srcdir
+
+if [[ "${target}" == *linux* ]]; then
+
+cd SFML-2.5.1/
+mv ./include $WORKSPACE/destdir/
+mv ./lib $WORKSPACE/destdir/
+mv ./share $WORKSPACE/destdir/
```

Comment:
```suggestion
    cd SFML-2.5.1/
    mv ./include ./lib ./share ${prefix}/.
```
But why don't you build this from source?

Reply:
It depends on https://github.com/kcat/openal-soft which hasn't been built as a JLL package yet.

Reply:
> which hasn't been built as a JLL package yet.

Any particular reason for not doing it?

Reply:
No. I was planning to upgrade CSFML to use JLLs firstly. I'm looking into the details now, but it looks like OpenAL depends on many backends as well. The SFML [doc](https://www.sfml-dev.org/tutorials/2.5/compile-with-cmake.php) just suggests using the binaries from Linux distributions, not sure how to configure those backends. I'll try it again tomorrow.